Add troubleshooting_mode_enabled session variable. (#14759)
cockroachdb/docs: _includes/v22.1/misc/session-vars.md

| <a name="transaction-rows-written-err"></a> `transaction_rows_written_err` | The limit for the number of rows written by a SQL transaction. If this value is exceeded the transaction will fail (or the event will be logged to `SQL_INTERNAL_PERF` for internal transactions). | `0` | Yes | Yes |
| <a name="transaction-rows-written-log"></a> `transaction_rows_written_log` | The threshold for the number of rows written by a SQL transaction. If this value is exceeded, the event will be logged to `SQL_PERF` (or `SQL_INTERNAL_PERF` for internal transactions). | `0` | Yes | Yes |
| <a name="transaction-status"></a> `transaction_status` | The state of the current transaction. See [Transactions](transactions.html) for more details. This session variable was called `transaction status` (with a space) in CockroachDB 1.x. It has been renamed for compatibility with PostgreSQL. | `NoTxn` | No | Yes |

> | <a name="troubleshooting_mode_enabled"></a> `troubleshooting_mode_enabled` | When enabled, avoid performing additional work on queries, particularly when the cluster is experiencing issues, unavailability, or failure. This session variable is used to avoid collecting and emitting telemetry data. | `off` | Yes | Yes |

Reviewer: Same suggestion as above

> | <a name="troubleshooting_mode_enabled"></a> `troubleshooting_mode_enabled` | When enabled, avoid performing additional work on queries, particularly when the cluster is experiencing issues, unavailability, or failure. This session variable is used to avoid collecting and emitting telemetry data. | `off` | Yes | Yes |

Author: ```suggestion
| <a name="troubleshooting_mode_enabled"></a> `troubleshooting_mode_enabled` | When enabled, avoid performing additional work on queries, such as collecting and emitting telemetry data. This session variable is particularly useful when the cluster is experiencing issues, unavailability, or failure. | `off` | Yes | Yes |
```
